webfonts/red-hat-font.css

```diff
@@ -1,213 +1,47 @@
-  font-weight: 700;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "RedHatDisplay";
-  src: url("./RedHatDisplay/RedHatDisplay-Black.woff2") format("woff2");
-  font-style: normal;
-  font-weight: 900;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "RedHatDisplay";
-  src: url("./RedHatDisplay/RedHatDisplay-BlackItalic.woff2") format("woff2");
-  font-style: italic;
-  font-weight: 900;
-  font-display: fallback;
-}
-
-/* RedHatText */
-
-@font-face {
-  font-family: "RedHatText";
-  src: url("./RedHatText/RedHatText-Light.woff2") format("woff2");
-  font-style: normal;
-  font-weight: 300;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "RedHatText";
-  src: url("./RedHatText/RedHatText-LightItalic.woff2") format("woff2");
-  font-style: italic;
-  font-weight: 300;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "RedHatText";
-  src: url("./RedHatText/RedHatText-Regular.woff2") format("woff2");
-  font-style: normal;
-  font-weight: 400;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "RedHatText";
-  src: url("./RedHatText/RedHatText-Italic.woff2") format("woff2");
-  font-style: italic;
-  font-weight: 400;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "RedHatText";
-  src: url("./RedHatText/RedHatText-Medium.woff2") format("woff2");
-  font-style: normal;
-  font-weight: 500;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "RedHatText";
-  src: url("./RedHatText/RedHatText-MediumItalic.woff2") format("woff2");
-  font-style: italic;
-  font-weight: 500;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "RedHatText";
-  src: url("./RedHatText/RedHatText-Bold.woff2") format("woff2");
-  font-style: normal;
-  font-weight: 700;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "RedHatText";
-  src: url("./RedHatText/RedHatText-BoldItalic.woff2") format("woff2");
-  font-style: italic;
-  font-weight: 700;
-  font-display: fallback;
-}
-
-/* RedHatMono */
-
-@font-face {
-  font-family: "RedHatMono";
-  src: url("./RedHatMono/RedHatMono-Light.woff2") format("woff2");
-  font-style: normal;
-  font-weight: 300;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "RedHatMono";
-  src: url("./RedHatMono/RedHatMono-LightItalic.woff2") format("woff2");
-  font-style: italic;
-  font-weight: 300;
-  font-display: fallback;
-}
-
 @font-face {
-  font-family: "RedHatMono";
-  src: url("./RedHatMono/RedHatMono-Regular.woff2") format("woff2");
+  font-family: 'RedHatDisplay';
+  src: url("./modified/RedHatDisplayVFModified.woff2") format('woff2-variations');
+  font-weight: 300 900;
   font-style: normal;
-  font-weight: 400;
   font-display: fallback;
 }
 
 @font-face {
-  font-family: "RedHatMono";
-  src: url("./RedHatMono/RedHatMono-Italic.woff2") format("woff2");
+  font-family: 'RedHatDisplay';
+  src: url("./modified/RedHatDisplayVF-ItalicModified.woff2") format('woff2-variations');
+  font-weight: 300 900;
```

Comment:
We've deleted `font-weight: 200` from line 3 above. I'm thinking that's intentional. 200 feels really thin but wanted to confirm.

Reply:
Correct, we're narrowing down the available weights to match what's currently used on redhat.com.